spec/features/password_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+describe User, type: :request do
+
+  describe 'post /users/password' do
```

Comment:
We don't need to test these emails since it has already been tested in the devise gem :). We need to overwrite the devise controller to render a JSON response, instead redirect to the default devise screens. WDYT @belgamo, @wenderjean?